package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.7.3",
+    "node-sass": "^4.14.1",
```

Comment:
If you choose scss please follow the rules of sass.

Reply:
ok